framework/src/loops/ComputeUserObjectsThread.C

```diff
@@ -271,6 +271,12 @@ ComputeUserObjectsThread::onInterface(const Elem * elem, unsigned int side, Boun
 
   for (const auto & uo : userobjs)
     uo->execute();
+
+  for (auto & uo : _domain_objs)
+  {
```

Comment:
Based on `interface_boundaries` parameter, maybe we can skip the call here? I might think too far here.